web_src/js/components/RepoActionView.vue

```diff
@@ -567,21 +662,81 @@ export function ansiLogToHTML(line) {
 
 .action-view-right {
   flex: 1;
-  color: var(--color-secondary-dark-3);
+  color: var(--color-console-fg-secondary);
   max-height: 100%;
   width: 70%;
   display: flex;
   flex-direction: column;
 }
 
+.action-view-right button {
+  background: transparent;
+  border: none;
+  color: var(--color-console-fg-secondary);
+}
+
+.action-view-right button:hover {
+  color: var(--color-console-fg);
+}
+
+.action-view-right .ui.dropdown .menu {
+  background: var(--color-console-menu-bg);
+  border-color: var(--color-console-menu-border);
+}
+
+.action-view-right .ui.dropdown .menu > .item {
+  color: var(--color-console-fg);
+}
+
+.action-view-right .ui.dropdown .menu > .item:hover {
+  color: var(--color-console-fg);
+  background: var(--color-console-hover-bg);
+}
+
+.action-view-right .ui.dropdown .menu > .item:active {
+  color: var(--color-console-fg);
+  background: var(--color-console-active-bg);
+}
+
+.action-view-right .ui.dropdown .menu > .divider {
+  border-top-color: var(--color-console-menu-border);
+}
+
+.action-view-right .ui.pointing.dropdown > .menu:not(.hidden)::after {
+  background: var(--color-console-menu-bg);
+  box-shadow: -1px -1px 0 0 var(--color-console-menu-border);
+}
```

Comment:
I have objection for these styles. Menu is menu, there shouldn't be a "console menu", it just makes the UI framework more difficult to maintain.

Reply:
It does follow GitHub and Drone's design though and a white menu in a dark box does look a bit out of place. I have been wanting to make the console light on light theme which would make this a non-issue, but there was pushpack against it.

Reply:
I can understand the "purpose", but the hacky approach doesn't look good to me. That's only my personal opinion, no block.

Reply:
It's just overriding the dropdown styles with a class prefix. I thought about overriding the CSS vars but ended up not because that can cause issues elsewhere in the box and I'm still of the opinion that CSS vars ideally only live on `:root` for easy querying via JS.

Reply:
I added some comments around the "override blocks" to make it visible which styles exactly override.